snc-library.sh

```diff
@@ -143,11 +143,13 @@ function create_json_description {
             | ${JQ} ".clusterInfo.openshiftVersion = \"${OPENSHIFT_RELEASE_VERSION}\"" \
             | ${JQ} ".clusterInfo.clusterName = \"${SNC_PRODUCT_NAME}\"" \
             | ${JQ} ".clusterInfo.baseDomain = \"${BASE_DOMAIN}\"" \
-            | ${JQ} ".clusterInfo.appsDomain = \"apps-${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\"" >${INSTALL_DIR}/crc-bundle-info.json
+            | ${JQ} ".clusterInfo.appsDomain = \"apps.${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\"" >${INSTALL_DIR}/crc-bundle-info.json
     if [ ${bundle_type} == "snc" ] || [ ${bundle_type} == "okd" ]; then
         openshiftInstallerVersion=$(${OPENSHIFT_INSTALL} version)
         tmp=$(mktemp)
-        cat ${INSTALL_DIR}/crc-bundle-info.json | ${JQ} ".buildInfo.openshiftInstallerVersion = \"${openshiftInstallerVersion}\"" \
+        cat ${INSTALL_DIR}/crc-bundle-info.json \
+            | ${JQ} ".buildInfo.openshiftInstallerVersion = \"${openshiftInstallerVersion}\"" \
+            | ${JQ} ".clusterInfo.appsDomain |= \"apps-${SNC_PRODUCT_NAME}.${BASE_DOMAIN}\"" \
```

Comment:
is |= mandatory? In my testing `=` seems to be enough? (`echo {} | jq ".clusterInfo.appsDomain=\"apps.crc.testing\"" |  jq ".clusterInfo.appsDomain=\"apps-crc.testing\""`

Reply:
As per man page of `jq` . For simple operation like this one `=` and `|=` works as same but update assignment is more precise for updating the existing value by keeping the same json structure. 
```
Update-assignment: |=
       This  is the "update" operator ´|=´. It takes a filter on the right-hand side and works out the new value for the property of . being assigned to by running the old value through this expression. For instance, (.foo, .bar) |= .+1 will build an object with the "foo" field set to the input´s "foo" plus 1, and the "bar" field set to the input´s "bar" plus 1.
```

Reply:
We simply want to overwrite the value, I don't think `|=` is needed, and it's not used anywhere else

Reply:
> We simply want to overwrite the value, I don't think |= is needed

As I said with current situation `=` and `|=` works same.

>and it's not used anywhere else

Doesn't mean we can't use it now. If that is adding confusion and block PR merge then I will update it.

Reply:
Updated.